# ui_pqExportStateWizard.h is not generated, therefore, it should not have to be installed.
# B. Hajgato Oct 28 2014
--- ParaView-v4.2.0-source.ORIG/Qt/Components/CMakeLists.txt	2014-09-23 16:46:44.000000000 +0200
+++ ParaView-v4.2.0-source/Qt/Components/CMakeLists.txt	2014-10-28 09:55:27.021704713 +0100
@@ -661,10 +661,10 @@
 #the pqSGExportStateWizard has subclasses that directly access
 #the UI file, and currently we don't have a clean way to break this hard
 #dependency, so for no we install this ui file.
-if(PARAVIEW_INSTALL_DEVELOPMENT_FILES)
-  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/ui_pqExportStateWizard.h"
-          DESTINATION "${VTK_INSTALL_INCLUDE_DIR}")
-endif()
+#if(PARAVIEW_INSTALL_DEVELOPMENT_FILES)
+#  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/ui_pqExportStateWizard.h"
+#          DESTINATION "${VTK_INSTALL_INCLUDE_DIR}")
+#endif()
 
 #IF (NOT PV_INSTALL_NO_DEVELOPMENT)
 #  # Headers
